src/stream/src/executor/sync_kv_log_store.rs

```diff
@@ -492,11 +723,22 @@ impl<S: StateStore> SyncedKvLogStoreExecutor<S> {
                             && buffer.no_flushed_items()
                         {
                             clean_state = true;
+                            self.metrics.clean_state.inc();
 
                             // Let write future resume immediately
                             if let WriteFuture::Paused { sleep_future, .. } =
                                 &mut write_future_state
                             {
+                                tracing::trace!("resuming paused future");
+                                if let Some(sleep_future) = sleep_future {
+                                    let deadline = sleep_future.deadline();
+                                    let now = Instant::now();
```

Comment:
The check here seems unnecessary. Neither the warning log nor the trace log seems to provide notable information.

Reply:
removed